Meetups/23/2025-02-20-minutes.md

```diff
@@ -0,0 +1,170 @@
+
+Andreas: We support semantic data harmonization to enrich basic contextual information.
+
+#### Natural Language Commands
+
+Andreas: we developed a way to command different devices using an open source LLM. The model is capable of choosing the right input for different actions just by using Thing Description data.
+
+Andreas: we can then develop an Agent framework to satisfy a certain goal.
+
+#### Demo
+
+Andreas: source code is [available](https://github.com/dashjoin/djapp-wot) and we have the corresponding [paper](https://github.com/dashjoin/djapp-wot/blob/main/paper.md).
+
+> Andreas shows the ui of the demo
+
+Andreas: voice to text support and code generation is supported. You can have dashboards too.
+
+### Question
+
+Ege: could you explain your process of getting to something working?
+
+Andreas: the platform is data driven. We start by loading data. You can create tables very easily once you have data in the DB. Everything has a page (much like a semantic Wikipedia). You can customize any page. Everything can be committed to github and personalized.
+
+Ege: Which opensource tools did you use? What are the tools and libraries that make it click?
+
+Andreas: thanks to standards and JSON schema.
+
+Henk: What discovery mechanism did you use? Very impressing presentation, and I can tell there were a lot of work.
+
+Andreas: To discover a new thing I will just input the Thing Description URL. In terms of the protocols we only support the easy stuff URL and JSON.
+
+Henk: Are you planning any "automatic loading" of TDs?
+
+Andreas: we have a registry CSV or a north bound API.
+
+Andreas: there are devices protected by password. We added a small extension to TD to express the role and the accounts who can use the device.
+
+Henk: If you have 5 people with different roles, do they access to different part of the system? or they can't do some actions?
+
+Andreas: on the database level there is a table which contains all the things and two other tables connected with Properties and Actions. We can lock different part of this table to different roles. there for we can support hiding or reviling Properties or Actions you can view and use.
+
+Andreas: we are interested working with the community and love to hear more from you. Please use our github repository. The project is at his beginning.
+
+Ege: any inputs for the standardization process?
+
+Andreas: we are still working of some of the feature. But from the research perspective is semantic harmonization. It would be wonderful to use JSON-LD and semantic more to automate this process.
+
+Vignesh: We there is something that we need to used but is not in the standard is it okay to be added in the TD?
+
+Andreas: we ignore them, but they don't show in the UI. You can make extension of the WoT manager to use this information.
```

Comment:
```suggestion
Andreas: We are still working on some of the features. But from the research perspective is semantic harmonization. It would be wonderful to use JSON-LD and semantic more to automate this process.

Vignesh: When there is something that we need to use but it is not in the standard, is it okay to be added in the TD?

Andreas: We ignore them, so they don't show in the UI. You can make an extension of the WoT manager to use this information.
```